app/actions/websocket/index.ts

```diff
@@ -172,6 +172,7 @@ export function doReconnect(now: number) {
             if (!me.error) {
                 if (isSupportedServerCalls) {
                     dispatch(loadCalls());
+                    dispatch(loadConfig(true));
```

Comment:
not blocking, but can this dispatches be batched?

Reply:
Yep, had to change up a bit, but it works. :)